### PR TITLE
Negative scenario: wrong sync code

### PR DESCRIPTION
**Pull request by Valentina1133**
Added negative scenario for syncing
Changed a bit scenario for syncing during onboarding
https://ethstatus.testrail.net/index.php?/cases/view/703631
Screenshot 2023-10-05 at 12 11 21

### CODE REVIEW

File: tests/test_syncing.py [hunk 1]
```diff
@@ -73,3 +88,18 @@ def test_sync_device_during_onboarding(multiple_instance, user_data):
...
+@allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/703631', 'Wrong sync code')
+@pytest.mark.case(703631)
+@pytest.mark.parametrize('wrong_sync_code', [
+    pytest.param('cs3:scrCo2f:Jni:mMDryn4AMHBoFTQfp9mPeVMaz4FiYfaCfpRokUvvJNsM:BR5WPtAZLRXj1Xw21azbiVG25jSfAfvZH8nLo2o')
+])
+def test_wrong_sync_code(sync_screen, wrong_sync_code):
+    with step('Open sync code form'):
+        sync_view = sync_screen.open_enter_sync_code_form()
+
+    with step('Paste wrong sync code on second instance and check that error message appears'):
```
Comment (anastasiyaig) [inline tests/test_syncing.py:102]: there is no second instance started

Comment (anastasiyaig) [inline tests/test_syncing.py:102]: there is no second instance started

Comment (anastasiyaig) [inline tests/test_syncing.py:102]: there is no second instance started

**Comment by anastasiyaig** [comment]
@Valentina1133 what was the idea of the test? I can see it could be one of these:

1. when u check error messages having 1 instance started, then we might want to extend the test with other test data (not only expired code, perhaps short one, missing some required parts, etc)
2. when u open instance 1 - copy sync code from there, open instance 2 - try to enter incorrect sync code first and then enter correct one and sync devices (for example)

What do u think?

File: tests/test_syncing.py [hunk 2]
```diff
@@ -73,3 +88,18 @@ def test_sync_device_during_onboarding(multiple_instance, user_data):
                 lambda: user_canvas.is_user_image_contains(user.name[:2]),
                 configs.timeouts.UI_LOAD_TIMEOUT_MSEC
             )
+
+
+@allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/703631', 'Wrong sync code')
+@pytest.mark.case(703631)
+@pytest.mark.parametrize('wrong_sync_code', [
+    pytest.param('cs3:scrCo2f:Jni:mMDryn4AMHBoFTQfp9mPeVMaz4FiYfaCfpRokUvvJNsM:BR5WPtAZLRXj1Xw21azbiVG25jSfAfvZH8nLo2o')
```
Comment (anastasiyaig) [inline tests/test_syncing.py:96]: can we indeed use a string that is not a sync code? The one above looks as sync code :)

**Review by anastasiyaig — APPROVED** [review]

**Comment by anastasiyaig** [comment]
just small comment left

**Review by StateOf-Vlado — APPROVED** [review]

File: tests/test_syncing.py (hunk 2 above)
Comment (Valentina1133) [reply tests/test_syncing.py:96]: Actually I did it on purpose. Sync code with a wrong symbol In it. But of course I can add something different

Comment (Valentina1133) [reply tests/test_syncing.py:96]: Maybe I can add more to check - different sync codes (sync code with mistake in it, something which is not a sync code at all like one symbol or smth, maybe sth else)?

Comment (anastasiyaig) [reply tests/test_syncing.py:96]: i talked to Magnus about it - lets just replace the value with a random string and merge. We add more scenarios later, or expand this one some day

Comment (Valentina1133) [reply tests/test_syncing.py:96]: Ok. Will do this

**Pull request merged**